docs/csharp/programming-guide/concepts/linq/snippets/LinqExtensions.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace linq
+{
+    // <LinqExtensionClass>
+    public static class LINQExtension
+    {
+        public static double Median(this IEnumerable<double>? source)
+        {
+            var countOfElementsInTheSet = source?.Any() ?? false;
```

Comment:
The variable name feels like it would be an integer, while it's boolean.
Probably `containsElements` or something like that be a better name?